### DISCUSSION TITLE
New filter option on /api/session end point

### DISCUSSION THREAD

**Discussion by giovannicarella** [devlikeapro/waha#299] [Ideas]
It would be useful to have the ability to filter the result of "/api/session" end point with a single session name.
With this feautures i'm able to know the state of the session.

**Comment by devlikepro** [comment]
Hi! Added in `2024.4.4` (WAHA Plus).

Make sure to update the image and restart the container.

https://waha.devlike.pro/docs/how-to/sessions/#get-session

----

### Get session
To get information about a specific session - call `GET /api/sessions/{session}`.
```json
{
  "name": "default",
  "status": "WORKING",
  "config": {
    "proxy": null,
    "webhooks": [
      {
        "url": "https://httpbin.org/post",
        "events": [
          "message",
          "session.status"
        ],
        "hmac": null,
        "retries": null,
        "customHeaders": null
      }
    ],
    "debug": false
  },
  "me": {
    "id": "79111111@c.us",
    "pushName": "WAHA"
  },
  "engine": {
    "engine": "NOWEB"
  }
}
```



[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by giovannicarella** [reply]
Fantastic... thanks a lot!
